src/entities/block-info-entity.ts

```diff
@@ -0,0 +1,106 @@
+import { Entity, Column, PrimaryColumn, ManyToOne, JoinColumn } from "typeorm";
+import { HeaderEntity } from "./header-entity";
+
+/*
+  Schema
+    header_id              VARCHAR(64) PRIMARY KEY REFERENCES node_headers (id),
+    timestamp              BIGINT  NOT NULL,
+    height                 INTEGER NOT NULL,
+    difficulty             BIGINT  NOT NULL,
+    block_size             INTEGER NOT NULL,
+    block_coins            BIGINT  NOT NULL,
+    block_mining_time      BIGINT,
+    txs_count              INTEGER NOT NULL,
+    txs_size               INTEGER NOT NULL,
+    miner_address          VARCHAR NOT NULL,
+    miner_reward           BIGINT  NOT NULL,
+    miner_revenue          BIGINT  NOT NULL,
+    block_fee              BIGINT  NOT NULL,
+    block_chain_total_size BIGINT  NOT NULL,
+    total_txs_count        BIGINT  NOT NULL,
+    total_coins_issued     BIGINT  NOT NULL,
+    total_mining_time      BIGINT  NOT NULL,
+    total_fees             BIGINT  NOT NULL,
+    total_miners_reward    BIGINT  NOT NULL,
+    total_coins_in_txs     BIGINT  NOT NULL,
+    max_tx_gix             BIGINT  NOT NULL,
+    max_box_gix            BIGINT  NOT NULL,
+    main_chain             BOOLEAN NOT NULL
+    PRIMARY KEY (header_id)
+*/
+
+@Entity({ name: "blocks_info" })
+export class BlockInfoEntity {
+  @PrimaryColumn({ name: "header_id" })
+  headerId!: string;
+
+  @Column({ name: "timestamp", type: "bigint" })
+  timestamp!: bigint;
+
+  @Column({ name: "height" })
+  height!: number;
+
+  @Column({ name: "difficulty", type: "bigint" })
+  difficulty!: bigint;
+
+  @Column({ name: "block_size" })
+  blockSize!: number;
+
+  @Column({ name: "block_coins", type: "bigint" })
+  blockCoins!: bigint;
+
+  @Column({ name: "block_mining_time", type: "bigint" })
+  blockMiningTime!: bigint;
```

Comment:
This field is nullable, so you need to add `nullable: true` to column settings and make prop nullable by replacing `!` by `?`